src/demo.js

```diff
@@ -141,6 +150,29 @@ class Demo extends React.Component {
                         </Cover>
                      </GridCell>
                      <GridCell>
+                        <Form
+                           initialValues={{ title: 'A Title', subtitle: 'A Subtitle', description: 'A Description' }}
+                           validation={{
```

Comment:
Yes good catch, forgot to update after making changes while using Communities. 081991b